bot/receiver.go

```diff
@@ -0,0 +1,128 @@
+package bot
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"strconv"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/subosito/gotenv"
+)
+
+func init() {
+	if err := gotenv.Load(); err != nil {
```

Comment:
Why not do this when we are initializing the rest of the bot stuff like database/etc? `init` stuff is generally for module setup and should be avoided as part of the application behavior.